.github/workflows/image-vscode-ssh-server.yml

```diff
@@ -0,0 +1,58 @@
+---
+name: Build VSCode SSH Server
+
+on: # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/image-vscode-ssh-server.yml
+      - images/vscode-ssh-server/**
+    tags:
+      - vscode-ssh-server@v*
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/workflows/image-vscode-ssh-server.yml
+      - images/vscode-ssh-server/**
+  release:
+    types:
+      - created
+
+jobs:
+  build-container:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      # Note: this must start with a number for the build to work correctly!
```

Comment:
This is copypasta probably...that was true for the Kairos stuff, but maybe not for this.